airflow/executors/base_executor.py

```diff
@@ -16,67 +14,86 @@
 # KIND, either express or implied.  See the License for the
 # specific language governing permissions and limitations
 # under the License.
-
+"""
+Base executor - this is the base class for all the implemented executors.
+"""
 from collections import OrderedDict
+from typing import Any, Dict, List, Optional, Set, Tuple, Union
 
-# To avoid circular imports
-import airflow.utils.dag_processing
-from airflow.configuration import conf
```

Comment:
I would have thought that we want this import, not `from airflow import conf`?

Reply:
I think it makes no real difference as we discussed before. And i actually (in the further commit) made a check to use it consistently 'from airflow import conf' and 'LoggingMixin' in as many places as possible (there are just a few exception where it matters - mainly around configuration itself and plugin manager.